iroha/Cargo.toml

```diff
@@ -16,6 +16,7 @@ bridge = []
 dex = []
 roles = ["iroha_data_model/roles"]
 network-mock = ["iroha_network/mock"]
+telemetry = ["iroha_telemetry", "lz4"]
```

Comment:
I would actually divide this into two cases and therefore features `dev-telemetry` for us (includes futures info), and just `telemetry` that we submit to a nice website about our network stats.